sdk/dapp-kit/src/hooks/useSuiClientQuery.ts

```diff
@@ -35,6 +40,34 @@ export type UseSuiClientQueryOptions<T extends keyof SuiRpcMethods, TData> = Par
 	'queryKey'
 >;
 
+export type GetSuiClientQueryOptions<T extends keyof SuiRpcMethods> = {
+	client: SuiClient;
+	network: string;
+	method: T;
+	options?: PartialBy<
+		Omit<UndefinedInitialDataOptions<SuiRpcMethods[T]['result']>, 'queryFn'>,
+		'queryKey'
+	>;
+} & (undefined extends SuiRpcMethods[T]['params']
+	? { params?: SuiRpcMethods[T]['params'] }
+	: { params: SuiRpcMethods[T]['params'] });
+
+export function getSuiClientQuery<T extends keyof SuiRpcMethods>({
```

Comment:
can we use this in `useSuiClientQuery` as well?

Reply:
Probably, I just didn't want to make a potentially breaking change yet